google-cloud-firestore/src/main/java/com/google/cloud/firestore/BulkWriterException.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.firestore;
+
+import com.google.cloud.firestore.BulkWriter.OperationType;
+import io.grpc.Status;
+
+/** The error thrown when a BulkWriter operation fails. */
+final class BulkWriterException extends FirestoreException {
+  /** @return The status code of the error. */
+  public Status getStatus() {
```

Comment:
nit: Java style usually has getters below constructors and constructors below fields.